website/src/pages/index.tsx

```diff
@@ -124,21 +124,20 @@ const testimonials: Testimonial[] = [
 ]
 
 export default ((props: any) => (
-    <Layout location={props.location}
+    <Layout
+        location={props.location}
         meta={{
-            title:
-                'Sourcegraph',
-            description:
-                'Start shipping better software faster with Universal Code Search.'
-            }}
+            title: 'Sourcegraph',
+            description: 'Start shipping better software faster with Universal Code Search.',
+        }}
     >
         <div className="home">
             <div className="home__intro container">
                 <div className="row justify-content-center">
                     <div className="col-lg-8 mb-6 mb-lg-0">
                         <h1 className="home__intro-header display-3">Universal Code Search</h1>
                         <p className="home__intro-text mt-3 font-weight-light">
-                            Google &amp; Facebook invested $100Ms to build internal developer&nbsp;platforms for{' '}
+                            Google &amp; Facebook invested $100Ms to build an internal&nbsp;tool for{' '}
```

Comment:
This may not matter, but didn't they create multiple tools to do this?
```suggestion
                            Google &amp; Facebook invested $100Ms to build internal&nbsp;tools for{' '}
```